build.gradle.kts

```diff
@@ -893,3 +893,7 @@ allprojects {
         }
     }
 }
+java {
+    sourceCompatibility = JavaVersion.VERSION_17
```

Comment:
We shouldn't be changing this as part of this patch. (I didn't notice it before.)

The rest looks good.

Reply:
Oops, good catch. For some reason, it's already second time when this change was added. First time I noticed and reverted it, second time I didn't. I wonder whether it's added by an IDE (I use IntelliJ IDEA) or by some build script.

Anyway, I removed the change. Could we please proceed?